apis/metal3.io/v1alpha1/baremetalhost_validation_test.go

```diff
@@ -263,6 +263,22 @@ func TestValidateCreate(t *testing.T) {
 			oldBMH:    nil,
 			wantedErr: "BMC driver libvirt does not support secure boot",
 		},
+		{
+			name: "InvalidBootMACAddress",
+			newBMH: &BareMetalHost{
+				TypeMeta:   tm,
+				ObjectMeta: om,
+				Spec: BareMetalHostSpec{
+					BMC: BMCDetails{
+						Address:         "irmc://127.0.1.1",
+						CredentialsName: "test1",
+					},
+					BootMACAddress: "00:00:00:00:00",
+					BootMode:       UEFISecureBoot,
+				}},
+			oldBMH:    nil,
+			wantedErr: "address 00:00:00:00:00: invalid MAC address",
```

Comment:
is it missing one more `00` at the end?
```suggestion
			wantedErr: "address 00:00:00:00:00:00: invalid MAC address",
```

Reply:
I think it was done specifically to make it invalid?

Reply:
Yes, it is deliberately incorrect for the test

Reply:
my bad disregard, I overlooked BootMACAddress on line 276